nondex-gradle-plugin/build.gradle

```diff
@@ -0,0 +1,42 @@
+apply plugin: 'groovy'
+apply plugin: 'maven'
+
+buildscript {
+    repositories {
+        maven {
```

Comment:
According to this [answer](https://discuss.gradle.org/t/what-is-difference-between-buildscript-classpath-and-dependencies-compile/4290/2) and the usage description of the [plugin-publish-plugin](https://plugins.gradle.org/docs/publish-plugin), I think we should leave it here.